### PR TITLE
Refactor processMeasurementTriple

### PR DESCRIPTION
**Pull request by alberth**
Bunch of simple refactor steps to convert the function into a more readable version

### CODE REVIEW

**Comment by alberth** [comment]
rebased to new master

**Comment by xlz** [comment]
Individual commits have really messy diffs so I didn't look.

I looked at the whole diff, and did not find any errors. Seems equivalent to me.

File: src/cpu_depth_packet_processor.cpp [hunk 1]
```diff
@@ -432,44 +432,53 @@ class CpuDepthPacketProcessorImpl: public WithPerfLogging
...
+            ir_image_b *= abMultiplierPerFrq;
+        }
+        float ir_amplitude = std::sqrt(ir_image_a * ir_image_a + ir_image_b * ir_image_b) * params.ab_multiplier;
 
-    m_out[0] = tmp3; // ir image a
-    m_out[1] = tmp4; // ir image b
-    m_out[2] = tmp5; // ir amplitude
+        m_out[0] = ir_image_a; // ir image a
+        m_out[1] = ir_image_b; // ir image b
+        m_out[2] = ir_amplitude; // ir amplitude
```
Comment (xlz) [inline src/cpu_depth_packet_processor.cpp]: These comments `// ir amplitude` are redundant now.

File: src/cpu_depth_packet_processor.cpp [hunk 2]
```diff
@@ -432,44 +432,53 @@ class CpuDepthPacketProcessorImpl: public WithPerfLogging
...
+        m_out[1] = 0;
+        m_out[2] = 65535.0; // some kind of norm calculated from ir_image_a and ir_image_b
+      }
+    }
+    else
+    {
+      // invalid pixel because zmultiplier < 0 ??
+      m_out[0] = 0; // ir image a
+      m_out[1] = 0; // ir image b
+      m_out[2] = 0; // ir amplitude
```
Comment (xlz) [inline src/cpu_depth_packet_processor.cpp]: Here too.

File: src/cpu_depth_packet_processor.cpp [hunk 3]
```diff
@@ -432,44 +432,53 @@ class CpuDepthPacketProcessorImpl: public WithPerfLogging
...
+        m_out[0] = ir_image_a; // ir image a
+        m_out[1] = ir_image_b; // ir image b
+        m_out[2] = ir_amplitude; // ir amplitude
+      }
+      else
+      {
+        // invalid pixel because saturated?
+        m_out[0] = 0;
+        m_out[1] = 0;
+        m_out[2] = 65535.0; // some kind of norm calculated from ir_image_a and ir_image_b
```
Comment (xlz) [inline src/cpu_depth_packet_processor.cpp]: `// some kind of norm calculated from ir_image_a and ir_image_b`

This comment is obsoleted by the expression for the actual `ir_amplitude`.

**Comment by alberth** [comment]
Individual commits are very small refactor steps that I did to ensure I wouldn't break current behaviour.
I can see it would seem quite messy, I'll have to find a better way.

For this branch, I additionally removed or shortened most comments, eliminated cond0 (replacing it with its value), and renamed "cond1" to "saturated". Finally, merged all commits into 1 "refactor" patch.

**Comment by xlz** [comment]
Manually merged.

**Pull request closed without merging**